### ISSUE TITLE
Cannot scan nav items in "tap for menu" when VO is on

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#185]
**Scenario to reproduce**

When VO is on and you tap to open the menu within the Shield VO cannot scan the elements, you need to know exactly where to tap to find items like language, province change, etc.

**Expected result**

VO should be able to scan all elements when dragging a finger (gesture) on points on the screen.

**Bug or issue / WCAG**

2.5.1: Pointer Gestures 
[https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html](url)

**Screenshots where available**

![Image from iOS](https://user-images.githubusercontent.com/35537308/85854243-06723180-b782-11ea-9354-2bf74417da7b.png)

**Device + OS**

iPhone 8 | 13.5.1

**Comment by JuliannaR** [comment]
Cross linked to CovidShield/mobile#170

**Comment by timarney** [comment]
@henrytao-me we can revisit or include this in the bottom sheet drawer menu fixes

**Comment by henrytao-me** [comment]
Ah ha. Another related issue 😄  👍

**Comment by JuliannaR** [comment]
Merged into this issue: https://github.com/cds-snc/covid-shield-mobile/issues/432
